system-reinstall-bootc/src/prompt.rs

```diff
@@ -94,7 +94,11 @@ pub(crate) fn get_ssh_keys(temp_key_file_path: &str) -> Result<()> {
         .flat_map(|user| &user.authorized_keys)
         .collect::<Vec<&PublicKey>>()
         .into_iter()
-        .map(|key| key.to_key_format() + "\n")
+        .map(|key| {
+            let mut key_copy = key.clone();
+            key_copy.options = None;
```

Comment:
Also super minor nit, while it's in the commit message I think this change is deserving of a comment too. i.e. just copy-paste your commit message here with a `#`